secretmanager/snippets/snippets_test.py

```diff
@@ -63,19 +63,64 @@ def iam_user() -> str:
     return "serviceAccount:" + os.environ["GCLOUD_SECRETS_SERVICE_ACCOUNT"]
 
 
+@retry.Retry()
+def retry_client_secret_path(
+    client: secretmanager.SecretManagerServiceClient,
+    project_id: str,
+    secret_id: str,
+) -> str:
+    # Retry to avoid 503 error & flaky issues
+    return client.secret_path(project_id, secret_id)
```

Comment:
Does it even do any network requests? Sounds like just formatting that cannot fail.

Reply:
Good point. Cleaning it up.